frontend/public/config.yml

```
@@ -554,4 +554,17 @@ collections:
+  - name: 'anouncement'
+    label: 'Anouncements'
+    folder: 'frontend/src/content/anouncement'
+    preview_path: k-vsite-git-cms-anouncement{{slug}}-froloehs-projects.vercel.app?via=pr-comment-visit-preview-link&passThrough=1
+    create: true
+    slug: '{{title}}'
+    fields:
+      - { label: 'Title', name: 'title', widget: 'string' }
+      - { label: 'Text', name: 'text', widget: 'string' }
+      - { label: 'Link', name: 'link', widget: 'string' }
+      - { label: 'Link text', name: 'link_text', widget: 'string' }
+      - { label: 'Active', name: 'active', widget: 'boolean' }
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)